I'm running into a failed assertion when trying to use child_process.send through cluster